README.md

```diff
@@ -12,38 +12,92 @@ The `amazon_cloud_code_generator` relies on the CloudFormation client. Hence, th
 
 ## Usage
 
-The [``amazon.cloud collection``](https://github.com/ansible-collections/amazon.cloud) modules are built using:
-
-```tox -e refresh_modules```
-
-The modules will be generated in the `cloud` subdirectory by default. A different target directory can be specified by using:
-
-```tox -e refresh_modules --target-dir /somewhere/else```
+The `amazon_cloud_code_generator`'s core content is hosted in the `amazon_cloud_code_generator/` directory containing the following subdirectories:
+
+- `cmd/` - hosts the main logic of the `amazon_cloud_code_generator`.
+    ```
+    │   ├── generator.py  # Implements the logic to scrape the CloudFormation resource schema and performs all the pre-processing needed to produce each module's DOCUMENTATION block
+    │   ├── refresh_examples.py  # Implements the logic to update EXAMPLES block scraping all the tasks tagged with `tags: [docs]` from `amazon_cloud_code_generator/data/tests/integration/targets/`
+    │   ├── refresh_modules.py  # Implements the logic to generate all the modules specified inside `resources.py` including all the utilities
+    │   ├── resources.py  # Sets the AWS resource names we are going to generate  
+    │   ├── utils.py  # Implements utility functions for the generator
+    ```
+
+- `config/` - Contains the `modules.yaml` file where, for each of the AWS resources we generate, an entry representing the module's name must be added. Additionally, you can also set other information that will be used in module's DOCUMENTATION like short_description, description, and so forth.
+
+- `data/` - Contains the content that will be copied over the generated collection. A high level overview of the subdirectory's content is shown below.
+    ```
+    ├── changelogs  # Changelog fragments that will be used for the new collection release
```

Comment:
I think changelogs and plugins folders should be at the same level.

Reply:
I do not see the 'changelogs' folder under data. Am I missing something?

Reply:
I fixed the indentation level. You're right. changelogs folder was one my local copy. Just removed from the tree.